content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+
+Finally, let’s consider allocation since the graphs above have only covered usage. Every sidecar resource has a default request of 100 millicores vCPU and 128 MB memory, as well as limits set for 2 vCPUs and 1 GB memory. For simplicity, we assume ztunnels and waypoint proxies have similar requests and limits as their sidecar counterparts. Breaking it down, that’s a 98% reduction in allocated resources with ztunnel, and 90% when waypoints are included.
+
+{{< image width="100%"
+    link="allocation.png"
+    caption="Memory usage by pod for sidecar and ambient pods"
+    >}}
+
+Going further, we can calculate a dollar amount for these numbers by referring to [GCP monthly pricing](https://cloud.google.com/compute/vm-instance-pricing) for a monthly cost per GB memory and per CPU. Consider these two different machines and their costs:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-4` (4 CPU, 32 GB) at $191/month
+
+Calculating the difference in memory results in 1GB costing $3.33/month. Similarly for CPU:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-2` (2 CPU, 16 GB) at $95/month
+
+Calculating the difference results in 1 CPU costing approximately $23/month. We can literally put a dollar amount on the savings ambient brings!
+
+## Test It Out
+
+In comparison to what most users run Istio with in production, this test cluster is tiny. However, we expect even more savings with larger clusters and when more services are deployed. We encourage everyone to see what the savings with Istio ambient mesh looks like in their environments. Also note that these scripts have been pushed to GitHub so feel free to check them out [here](https://github.com/solo-io/ambient-performance/tree/boutique-demo). For tracking CPU and memory usage throughout the test scenarios, versions of [Prometheus](https://prometheus.io/), [node-exporter](https://prometheus.io/docs/guides/node-exporter/), and [Grafana](https://grafana.com/) are installed. A custom Grafana dashboard was created for observing relevant data, which can be found and imported from GitHub [here](https://github.com/solo-io/ambient-performance/blob/boutique-demo/dashboard/ambient-performance-analysis.json).
+
+## Conclusion
+
+Ka-ching. These results were collected with an alpha version of ambient, which is now [merged into the main branch](/blog/2023/ambient-merged-istio-main/) and included in Istio 1.18.
+
+Huge thanks to all the Istio maintainers who work very hard to make ambient as lean and performant as possible, ambient service mesh’s goal of reducing infrastructure costs is bearing fruit and setting a solid foot forward on its roadmap to production readiness. These early numbers suggest users could cut their cloud usage by *99%* and resource requirements by *90%* – especially if users only require an L4 mesh.
```

Comment:
You had up to 99% savings in usage and 98% in allocation earlier, while you have "cut their cloud usage by *99%* and resource requirements by *90%*" here, can you please be consistent?

Reply:
Think this is clarified